### PR TITLE
release v0.4.0

### PR DESCRIPTION
**Pull request by andreeaflorescu**
For details about this release please check the CHANGELOG.md file.

### CODE REVIEW

**Review by AlexandruCihodaru — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# [v0.4.0]
+
+## Fixed
+
+- Fixed potential overflow in calls to `align_up`.
```
Comment (lauralt) [inline CHANGELOG.md:5]: I think we have an issue for this one as well: https://github.com/rust-vmm/linux-loader/issues/66.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+- [[#62]](https://github.com/rust-vmm/linux-loader/issues/62) The
+  `load_cmdline` function now takes as a parameter the crate defined
+  `Cmdline` object instead of `Cstr`. This means that customers don't need to
+  convert the object before calling into `load_cmdline`.
+- [[#83]](https://github.com/rust-vmm/linux-loader/issues/83) Updated the
+  vm-memory dependency requirement to the latest version (0.6.0).
+
+## Added
+
+- Implemented `From<Cmdline>` for `Vec<u8>`. This replaces the obsolete `Into`
```
Comment (lauralt) [inline CHANGELOG.md:18]: I think I would link the PR for things that don't have an issue.

**Comment by lauralt** [comment]
Do we update the version in a separate PR: https://github.com/rust-vmm/linux-loader/blob/main/Cargo.toml#L3?

**Comment by andreeaflorescu** [comment]
> Do we update the version in a separate PR: https://github.com/rust-vmm/linux-loader/blob/main/Cargo.toml#L3?

Nop, I just forgot to add the file to git index :laughing:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+- [[#62]](https://github.com/rust-vmm/linux-loader/issues/62) The
+  `load_cmdline` function now takes as a parameter the crate defined
+  `Cmdline` object instead of `Cstr`. This means that customers don't need to
+  convert the object before calling into `load_cmdline`.
+- [[#83]](https://github.com/rust-vmm/linux-loader/issues/83) Updated the
+  vm-memory dependency requirement to the latest version (0.6.0).
+
+## Added
+
+- [[#79]](https://github.com/rust-vmm/linux-loader/pull/79) Implemented
```
Comment (lauralt) [inline CHANGELOG.md:19]: Should this actually be part of the Changed section? :thinking:

**Review by lauralt — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (andreeaflorescu) [reply CHANGELOG.md:19]: The `From` implementation is new, that's why I added it to this section. Nothing changes because `into` can still be called.

**Review by alexandruag — APPROVED** [review]

**Pull request merged**